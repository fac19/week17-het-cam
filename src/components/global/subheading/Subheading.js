import styled from "styled-components";
import { fontColour } from "../colours";

const Subheading = styled.h2`
  font-family: "Josefin Sans", sans-serif;
  font-size: 1.5rem;
  color: ${fontColour};
  text-align: center;
  text-transform: uppercase;

  @media (min-width: 600px) {
    font-size: 2.2rem;
  }
`;

export default Subheading;
