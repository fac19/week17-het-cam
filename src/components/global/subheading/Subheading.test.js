import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Subheading from "./Subheading";

test("Heading component is styled", () => {
  const tree = renderer.create(<Subheading />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("color", "#032049");
  expect(tree).toHaveStyleRule("font-size", "1.5rem");
  expect(tree).toHaveStyleRule("font-size", "2.2rem", {
    media: "(min-width:600px)",
  });
});
