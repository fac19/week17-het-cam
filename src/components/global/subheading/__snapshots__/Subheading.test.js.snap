// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component is styled 1`] = `
.c0 {
  font-family: "Josefin Sans",sans-serif;
  font-size: 1.5rem;
  color: #032049;
  text-align: center;
  text-transform: uppercase;
}

@media (min-width:600px) {
  .c0 {
    font-size: 2.2rem;
  }
}

<h2
  className="c0"
/>
`;
